src/chrome/content/rules/SixthTone.com.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	No working URL known:
+		file.sixthtone.com
+		image1.sixthtone.com
+		image2.sixthtone.com
+		image3.sixthtone.com
+		interaction.sixthtone.com
```

Comment:
@Bisaloo Just search for `site:interaction.sixthtone.com`

Reply:
@J0WI, I'll be busy in the next couple of days but I'll try to finish all my pending PRs after that. Thank you!